packages/fxa-auth-server/lib/senders/emails/partials/support/en.ftl

```diff
@@ -1,3 +1,3 @@
 # Variables:
-#  $supportUrl (String) - Link to https://accounts.firefox.com/support
+#  $supportUrl (String) - Link to https://support.mozilla.org/kb/im-having-problems-my-firefox-account
 support-message = For more information, please visit { $supportUrl }
```

Comment:
If we are not deleting the support partial, `support-message-2 = For more info, visit { -brand-mozilla } Support: { $supportUrl }.`